extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -65,3 +103,70 @@ export const renderPageAttachmentUI = (pageEl, attachmentEl) => {
   textEl.textContent = openLabel;
   return openAttachmentEl;
 };
+
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @return {!Element}
+ */
+const renderInlinePageAttachmentUI = (win, pageEl, attachmentEl) => {
+  const openAttachmentEl = buildOpenInlineAttachmentElement(pageEl);
+
+  const openLabelAttr = attachmentEl.getAttribute('cta-text');
+  if (openLabelAttr) {
+    const textEl = win.document.createElement('span');
+    textEl.classList.add("i-amphtml-story-inline-page-attachment-label");
+    const openLabel = openLabelAttr && openLabelAttr.trim();
+    textEl.textContent = openLabel;
+    setImportantStyles(textEl, {
+      'display': 'block',
+    });
+    openAttachmentEl.appendChild(textEl);
+  }
+
+  const openImgAttr = attachmentEl.getAttribute('cta-img');
+
+  const ctaImgEl = openAttachmentEl.querySelector(
+    '.i-amphtml-story-inline-page-attachment-img'
+  );
+
+  const openImgAttrAlt = attachmentEl.getAttribute('cta-img-alt');
+  if (openImgAttrAlt) {
+    ctaImgEl.setAttribute("aria-label", openImgAttrAlt);
+  }
+
+  setImportantStyles(ctaImgEl, {
+    'background-image': 'url(' + openImgAttr + ')',
+  });
+
+  const openImgAttr2 = attachmentEl.getAttribute('cta-img-2');
+
+  if (openImgAttr2) {
+    const ctaImgEl2 = win.document.createElement('span');
+    ctaImgEl2.classList.add("i-amphtml-story-inline-page-attachment-img");
+    setImportantStyles(ctaImgEl2, {
+      'background-image': 'url(' + openImgAttr2 + ')'
+    });
+    const openImgAttrAlt2 = attachmentEl.getAttribute('cta-img-2-alt');
+    if (openImgAttrAlt2) {
+      ctaImgEl2.setAttribute("aria-label", openImgAttrAlt2);
```

Comment:
These actually won't be read if the parent element has an `aria-label` attribute. 
I think these can be considered decorative and therefore we don't need to announce them. 
We can remove this attribute logic. I didn't realize this until testing it on a device, it's definitely a good idea to test a11y on a device while developing for something like this.